pkg/config/mesh/mesh_test.go

```diff
@@ -91,6 +89,11 @@ defaultConfig:
 	}
 	// Verify overrides
 	got, err = mesh.ApplyMeshConfigDefaults(`
+serviceSettings: 
+  - settings:
+      clusterLocal: true
+    host:
+      - "*.myns.svc.cluster.local"
```

Comment:
should we extend validatemeshConfig?

Reply:
specifically that its a valid host - `foo.*` is not legal right?

Reply:
ah good point.